### DISCUSSION TITLE
How to connect One Resource to another

### DISCUSSION THREAD

**Discussion by sohailahmad07** [filamentphp/filament#14516] [Help]
### Package

Panel builder

### Package Version

3.2.115

### How can we help you?

Hi everyone,

- I’m currently working on an application that has two main resources: User and Invoices. I need assistance with the following functionality:
- Navigating from User to Invoices: When I click on a user record in the User resource, it should direct me to the Invoices section, displaying only the invoices related to that specific user.
- Creating/Editing Invoices: When I create or edit an invoice, the application should remember the user context, ensuring that the invoice is linked to the correct user.
- Access Control: I have disabled the global invoices view, so the only way to access invoices is through the user profile.

I’d appreciate any guidance or best practices on implementing this functionality effectively. Thanks in advance!

**Comment by leandrocfe** [comment]
Why not a [relation manager](https://filamentphp.com/docs/3.x/panels/resources/relation-managers#creating-a-relation-manager)?
